cypress/integration/accessibility.spec.js

```diff
@@ -1,95 +1,175 @@
+    cy.contains('h1', 'Upload a document to');
+
+    // select document
+    cy.fixture('test.pdf').as('document');
+    cy.get('[data-testid="file-upload-input"]').then(function (el) {
+      const blob = Cypress.Blob.base64StringToBlob(
+        this.document,
+        'application/pdf'
+      );
+
+      const file = new File([blob], 'document.pdf', {
+        type: 'application/pdf',
+      });
+      const list = new DataTransfer();
+
+      list.items.add(file);
+      const myFileList = list.files;
+
+      el[0].files = myFileList;
+      el[0].dispatchEvent(new Event('change', { bubbles: true }));
+    });
+
+    // enter the document type
+    cy.contains('.usa-radio', 'Awarded VPAT')
+      .find('input')
+      .check({ force: true });
+
+    // click upload button
+    cy.get('[data-testid="Upload-Document"]').click();
+
+    cy.contains('.usa-alert', 'document.pdf uploaded to');
+    // verify that the document is in the list
+
+    cy.get('[data-testid="AccessibilityDocumentsList"] tbody tr')
+      .should('have.length', 1)
+      .first()
+      .within(() => {
+        cy.contains('Awarded VPAT');
+        cy.contains('Virus scan in progress...');
+      });
+
+    // view document
+    // remove document
+  }
+
+  it('can create a request and see its details', () => {
+    cy.localLogin({ name: 'A11Y' });
+    create508Request(cy);
+
     cy.get('.accessibility-request__side-nav').within(() => {
-      cy.contains('h2', 'Test Dates and Scores')
+      cy.contains('h2', 'Test Dates and Scores');
       cy.get('.accessibility-request__other-details').within(() => {
-        cy.contains('dt', 'Submission date')
-        const dateString = formatDate(new Date().toISOString())
-        cy.contains('dl', dateString)
+        cy.contains('dt', 'Submission date');
+        const dateString = formatDate(new Date().toISOString());
+        cy.contains('dl', dateString);
+
+        cy.contains('dt', 'Business Owner');
+        cy.contains('dl', 'Shane Clark, OIT');
+
+        cy.contains('dt', 'Project name');
+        cy.contains('dl', 'TACO');
 
-        cy.contains('dt', 'Business Owner')
-        cy.contains('dl', 'Shane Clark, OIT')
+        cy.contains('dt', 'Lifecycle ID');
+        cy.contains('dl', '000000');
+      });
 
-        cy.contains('dt', 'Project name')
-        cy.contains('dl', 'TACO')
+      cy.contains('a', '508 testing templates (opens in a new tab)');
+      cy.contains('a', 'Steps involved in 508 testing (opens in a new tab)');
+      cy.contains('button', 'Remove this request from EASi');
+    });
+  });
+
+  it('adds a document from a 508 request as the owner', () => {
+    cy.localLogin({ name: 'CMSU' });
+    create508Request(cy);
+    addAndRemoveDocument(cy);
+  });
 
-        cy.contains('dt', 'Lifecycle ID')
-        cy.contains('dl', '000000')
-      })
+  it('adds and removes a document from a 508 request as an admin', () => {
+    cy.localLogin({ name: 'CMSU' });
+    create508Request(cy);
```

Comment:
Since we've already tested the flow to create an accessibility request, can we include an existing request through the db seed so that we don't need to test that here?  That in the interest of speeding up the tests.

Reply:
I agree that would be nicer, and a good bit of followup work. The seeds are kind of mess right now, and since these tests need to be run on accessibility requests without any other documents currently, just making a new one seemed liked the simplest way to ensure that they had their own clean request to work from.

Longer term I do think we need to be able to lean more on seeding for exactly the reason you mention.